docs/posts/2025/08/interactive-shell.md

```diff
@@ -32,28 +32,31 @@ asciinema(
   cols=108,
   rows=25,
   markers=[
-    [0, "Start installation"],
-    [11, "Build failure"],
-    [15, "Enter interactive debug environment"],
-    [18, "Repeat failed command"],
-    [23, "Explore environment"],
-    [32, "Exit environment"]
+    [0, "View EasyBuild version"],
+    [10, "Start installation"],
+    [21, "Build failure"],
+    [35, "Enter interactive debug environment"],
+    [38, "Repeat failed command"],
+    [42, "Explore environment"],
+    [57, "Exit environment"]
```

Comment:
The markers are wrong when I check the rendered version, these seem correct:
```suggestion
    [23, "Enter interactive debug environment"],
    [26, "Repeat failed command"],
    [28, "Explore environment"],
    [40, "Exit environment"]
```

but this doesn't agree with the times in the `.cast` file... 🤔